### PR TITLE
[irods/irods#7421] irm: expand docs force option (4-3-stable)

### PR DESCRIPTION
**Pull request by stsnel**
Note that this option makes irm ignore non-existent data objects.

### CODE REVIEW

File: src/irm.cpp [hunk 1]
```diff
@@ -122,6 +122,8 @@ usage() {
         " ",
         "Options are:",
         " -f  force - Immediate removal of data objects without putting them in trash.",
+        "             The command does not show an error message or return a non-zero ",
+        "             status code for non-existent data objects.",
```
Comment (korydraughn) [inline src/irm.cpp:126]: This is slightly incorrect (based on the implementation).

How about ...
```
Ignores non-existent data objects and collections.
```

**Review by korydraughn — COMMENTED** [review]
Thanks for the contribution.

Just one tweak and we'll get this added.

File: src/irm.cpp (hunk 1 above)
Comment (stsnel) [reply src/irm.cpp]: Thank you, that is indeed better. I've changed the text.

**Comment by korydraughn** [comment]
The issue number linked cannot be used because it's already part of a closed milestone.

I've created irods/irods#7421 for this work. Please use that.

**Review by alanking — APPROVED** [review]

**Pull request merged**